docs/content/get-started/generate-providers.md

```diff
@@ -0,0 +1,199 @@
+---
+title: "Generate the providers"
+weight: 1
+aliases:
+  - /docs/getting-started/setup
+  - /getting-started/setup
+  - /docs/getting-started/generate-providers
+  - /getting-started/generate-providers
+---
+
+
+# Generate `google` and `google-beta` providers
+
+This quickstart guides you through setting up your development environment, making a change to `magic-modules`, generating provider changes to the `google` and `google-beta` Terraform providers, and running tests related to the change.
+
+## Before you begin
+
+1. [Install the gcloud CLI.](https://cloud.google.com/sdk/docs/install)
+1. In the Google Cloud console, on the project selector page, select or [create a Google Cloud project](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
+   {{< hint info >}}
+   **Note:** If you don't already have a project to use for testing changes to the Terraform providers, create a project instead of selecting an existing poject. After you finish making your changes, you can delete the project, removing all resources associated with the project.
```

Comment:
`removing` should be `remove` ?

Reply:
I believe `removing` is correct - this is standard language used by quickstarts. See https://cloud.google.com/compute/docs/create-linux-vm-instance for example.

Reply:
I see. Thanks.